File: easy/ExcelSheetTitle.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder ans = new StringBuilder();
        while(columnNumber > 0){
            int col = columnNumber % 26;
            if(col == 0){
                ans.append('Z');
                columnNumber --;
            }
            else
            ans.append((char)('A' + col - 1));
            columnNumber =  columnNumber / 26;
        }
        return ans.reverse().toString();
    }
}

File: easy/ExcelSheetTitle.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string ans = "";
        while (columnNumber > 0) {
            int col = columnNumber % 26;
            if (col == 0) {
                ans.push_back('Z');
                columnNumber--;
            } else {
                ans.push_back('A' + col - 1);
            }
            columnNumber /= 26;
        }
        reverse(ans.begin(), ans.end());
        return ans;
    }
};